specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/cosmos-db.json

```diff
@@ -7873,6 +7909,62 @@
         }
       }
     },
+    "LocationGetResult": {
+      "description": "Cosmos DB location get result",
+      "type": "object",
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The unique resource identifier of the ARM resource."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the ARM resource."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The type of Azure resource."
+        },
+        "location": {
```

Comment:
if this is indeed a tracked resource, please ref it from common-types

Reply:
The common type is ARMResourceProperties which contains id, name,type, location, tages, and identity. For me, I don't need tags and identity. Should I use 
1. Explicitly define the subset I needed here? which is current implementation
2. Use ARMResourceProperties, but only assign values that I needed and assign empty to not used value(identity, tages)?

Reply:
supporting tags is a requirement for tracked resources https://armwiki.azurewebsites.net/rp_onboarding/tracked_vs_proxy_resources.html?q=tracked